core/src/main/java/tc/oc/pgm/projectile/ProjectileModule.java

```diff
@@ -54,20 +55,18 @@ public ProjectileModule parse(MapFactory factory, Logger logger, Document doc)
           XMLUtils.flattenElements(doc.getRootElement(), "projectiles", "projectile")) {
         String id = XMLUtils.getRequiredAttribute(projectileElement, "id").getValue();
         String name = projectileElement.getAttributeValue("name");
-        Double damage =
-            XMLUtils.parseNumber(
-                projectileElement.getAttribute("damage"), Double.class, (Double) null);
-        double velocity =
-            XMLUtils.parseNumber(
-                Node.fromChildOrAttr(projectileElement, "velocity"), Double.class, 1.0);
-        ClickAction clickAction =
-            XMLUtils.parseEnum(
-                Node.fromAttr(projectileElement, "click"), ClickAction.class, ClickAction.BOTH);
+        Double damage = XMLUtils.parseNumber(
+            projectileElement.getAttribute("damage"), Double.class, (Double) null);
+        double velocity = XMLUtils.parseNumber(
+            Node.fromChildOrAttr(projectileElement, "velocity"), Double.class, 1.0);
+        ClickAction clickAction = XMLUtils.parseEnum(
+            Node.fromAttr(projectileElement, "click"), ClickAction.class, ClickAction.BOTH);
         Class<? extends Entity> entity =
             XMLUtils.parseEntityTypeAttribute(projectileElement, "projectile", Arrow.class);
-        Float power =
-            XMLUtils.parseNumber(
-                Node.fromChildOrAttr(projectileElement, "power"), Float.class, (Float) null);
+        BlockMaterialData blockMaterial =
+            XMLUtils.parseBlockMaterialData(Node.fromAttr(projectileElement, "material"));
```

Comment:
only parse material if the entity type is a falling block, that way it will launch an unused node if you try to use it for smthg else